Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -2477,7 +2480,7 @@ $ adb shell cat /data/local/tmp/nowyouseeme
 ABCD
 ```
 
-It's time to write the kernel module. For file-hiding, you'll need to hook one of the system calls used to open (or check for the existence of) files. There are many of these—open, openat, access, accessat, facessat, stat, fstat, etc. For now, you'll only hook the openat system call.  This is the syscall the /bin/cat program uses when accessing a file, so the call should be suitable for a demonstration.
+It's time to write the kernel module. For file-hiding, you'll need to hook one of the system calls used to open (or check for the existence of) files. There are many of these—open, `openat`, `access`, `accessat`, `facessat`, `stat`, `fstat`, etc. For now, you'll only hook the `openat` system call.  This is the syscall the /bin/cat program uses when accessing a file, so the call should be suitable for a demonstration.
```

Comment:
```suggestion
It's time to write the kernel module. For file-hiding, you'll need to hook one of the system calls used to open (or check for the existence of) files. There are many of these: `open`, `openat`, `access`, `accessat`, `facessat`, `stat`, `fstat`, etc. For now, you'll only hook the `openat` system call.  This is the syscall that the /bin/cat program uses when accessing a file, so the call should be suitable for a demonstration.
```

Reply:
Agree, and I have a feeling that we should in general refrain from using dashes (—). What do you think? Also I've seen that the ones we have are marked as "bad-chars" by the "Bad Chars" plugin that I have installed for vscode. This might give us some headaches in the future autom. builds if we don't properly take care of them.